docs/components/modeler/web-modeler/connectors/available-connectors/slack.md

```diff
@@ -0,0 +1,92 @@
+---
+id: slack
+title: Slack connector
+description: Working with Slack connector
+---
+
+The **Slack connector** allows you to interact with a [Slack](https://slack.com/) workspace, for example by sending Slack messages from your BPMN processes.
```

Comment:
Minor comment, possible to update the text to the following?
"Slack connector allows you to send messages to channels or users in your Slack workspace from your BPMN process."

Reply:
Makes sense, updated